core/crypto/crypto.h

```diff
@@ -84,8 +84,6 @@ class Crypto : public Reference {
 };
 
 class ResourceFormatLoaderCrypto : public ResourceFormatLoader {
-	GDCLASS(ResourceFormatLoaderCrypto, ResourceFormatLoader);
```

Comment:
This is to be consistent with #29872.
It now seems to me that simply skipping them in DocData as done in this PR would have been sufficient, so it's not clear to me what difference `GDCLASS` makes in this case.

Reply:
A class without `GDCLASS` cannot be exposed, but the right way to do that is to just not call `ClassDB::register_*`.
That being said if the class doesn't need it, removing `GDCLASS` can be beneficial to reduce executable bloat, but it must be done with care as some classes depend on it (for `_bind_methods` for example). Also not sure how it affects inheritance, e.g.: `Object` <- `B` <- `C` where `C` has `GDCLASS` but `B` doesn't.